operations/munge.rst

```diff
@@ -10,9 +10,18 @@ key.
 
 The etcd server has authentication enabled by default, to protect the Munge
 key. In order to be able to retrieve the Munge key, an etcd account is
-required.  OSD provides Juju action ``etcd-create-munge-account`` on the
+required. OSD provides the Juju action ``etcd-create-munge-account`` on the
 ``slurmctld`` charm to automate the creation of such an account.
 
+.. TODO
```

Comment:
I'm not quite sure why this is a TODO...

Reply:
Implementation is not completed yet in the slurm-charms, so the docs are just a start. There are still some bits to figure out and I wanted to have some notes out of my head to not forget later.

Reply:
@dusktreader I upted the document with the complete text. This should have been marked as `draft` from the beginning, but I forgot to do that. The docs are now ready.